### DISCUSSION TITLE
Make a Jetbrain like key binding

### DISCUSSION THREAD

**Discussion by leodlplq** [zed-industries/zed#8078] [Feature Requests]
I tried to make a custom key binding that I like on Jetbrain to toggle file finder, it's `SHIFT` `SHIFT`, I tried putting it in the config file but can't make it work. 

Here's my `keymap.json`

```json
[
  {
    "context": "Editor",
    "bindings": {
      "shift shift": "file_finder::Toggle"
    }
  }
]
```

What am i doing bad ?

**Comment by SomeoneToIgnore** [comment]
You're not doing anything bad, just Zed not supporting double-tab keybinds yet: https://github.com/zed-industries/zed/issues/6460
